File: Week 6 - Python/pset6/mario/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if height > 0 and height <= 8:
        break

for i in range(height):
    for j in range(height):
        if (i + j) < height - 1:
            print(" ", end="")
        else:
            print("#", end="")
    print("  ", end="")
    for k in range(height):
        if i >= k:
            print("#", end="")
    print("")

File: Week 4 - Memory/lecture4/mario.c
#include <stdio.h>
#include <cs50.h>

void draw(int h);

int main (void)
{
    int height = get_int("Height: ");
    draw(height);
}

void draw(int h)
{
    if (h == 0)
    {
        return; // TO AVOID draw FUNCTION CALL ITSELF SEVERAL TIMES AND GET A SEGMENTATION FAULT.
    }

    draw(h - 1);

    for (int i = 0; i < h; i++)
    {
        printf("#");
    }
    printf("\n");
}